terraform/modules/jaspersoft/jaspersoft_install_bucket.tf

```diff
@@ -64,25 +64,29 @@ resource "aws_s3_object" "tomcat_systemd_service_file" {
   key     = "tomcat.service"
   content = local.tomcat_systemd_service_file_templated
   etag    = md5(local.tomcat_systemd_service_file_templated)
+  tags    = { environment = "shared" }
```

Comment:
Currently shared between test and staging, will make a separate bucket for prod, but this stops terraform thrashing between them